modules/gcp-secret/README.md

```diff
@@ -0,0 +1,57 @@
+## GCP Secret Manager Module
```

Comment:
this seems is not right place to have this module,
this module is part of google gcp and not any?